RetailCoder.VBE/Inspections/QuickFixes/AssignedByValParameterMakeLocalCopyQuickFix.cs

```diff
@@ -29,6 +31,7 @@ public AssignedByValParameterMakeLocalCopyQuickFix(Declaration target, Qualified
             _dialogFactory = dialogFactory;
             _parserState = parserState;
             _forbiddenNames = parserState.DeclarationFinder.GetDeclarationsWithIdentifiersToAvoid(target).Select(n => n.IdentifierName);
+            _setRequirementEvaluator = new VariableRequiresSetAssignmentEvaluator(_parserState);
         }
 
         public override bool CanFixInModule => false;
```

Comment:
I was thinking that because the user selected to modify one case of ByVal assignment with a local copy, it would be too much to assume that it would be appropriate to apply the fix to every case without the user making a decision about the intent for each case.